src/correlation.ts

```diff
@@ -4,17 +4,17 @@ import { createHash } from "node:crypto";
 const OPTIONAL_VARIABLES = ["INVOCATION_ID"];
 
 export type AnonymizedCorrelationHashes = {
-  correlation_source: string;
+  correlationSource: string;
   repository?: string;
   run?: string;
-  run_differentiator?: string;
+  runDifferentiator?: string;
   workflow?: string;
   groups: Record<string, string | undefined>;
 };
 
 export function identify(projectName: string): AnonymizedCorrelationHashes {
   const ident = {
-    correlation_source: "github-actions",
+    correlationSource: "github-actions",
```

Comment:
these need to be _'d because the server expects them to be _'d

Reply:
(fwiw, you can use serde to add aliases so that both `correlation_source` and `correlationSource` are accepted)

Reply:
Open to suggestion but I'm going to use snake case here and attach a comment (and also turn on camel case checking in ES Lint, plus appropriate ignore statements).